scripts/esm-support/supportESM.js

```diff
@@ -0,0 +1,61 @@
+
+let fs = require('fs-extra');
```

Comment:
can delete this file and its corresponding script target from package.json
now that it's in the plop template and the icons were simplified with the pattern matching in exports, this has become unnecessary